core/app/models/spree/store.rb

```diff
@@ -64,7 +64,7 @@ class Store < Spree::Base
 
     validates :digital_asset_authorized_clicks, numericality: { only_integer: true, greater_than: 0 }
     validates :digital_asset_authorized_days, numericality: { only_integer: true, greater_than: 0 }
-    validates :code, uniqueness: { conditions: -> { with_deleted } }
+    validates :code, uniqueness: { case_sensitive: true, conditions: -> { with_deleted } }
```

Comment:
```suggestion
    validates :code, uniqueness: { case_sensitive: false, conditions: -> { with_deleted } }
```

this needs to stay false, we're doing lookups by code in several places and it's based on the URL